libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -74,12 +86,15 @@ library
         bytestring,
         cardano-data,
         cardano-crypto-class,
+        cardano-crypto-wrapper,
+        cardano-crypto,
         cardano-ledger-allegra,
         cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib},
         cardano-ledger-alonzo-test,
         cardano-ledger-api,
         cardano-ledger-babbage:{cardano-ledger-babbage, testlib},
         cardano-ledger-babbage-test,
+        cardano-ledger-byron,
```

Comment:
Why do we need this dependency on `cardano-ledger-byron` and on `cardano-crypto*` packages above?

Reply:
Because we are stuck with Byron Adresses forever, I felt I needed to add tests that had Byron addresses. To generate Byron Verification and Singing key pairs, I need these two modues.
import qualified Cardano.Chain.Common as Byron
import qualified Cardano.Crypto.Signing as Byron
Both if there are in  'cardano-ledger-byron' so that is why   cardano-ledger-byron is in the 'build depends' of cardano-ledger-test.